### ISSUE TITLE
I would like to convert Milliliter to liter (1000)

### ISSUE THREAD

**Issue by yesparthasarathy** [pizzafox/convert#105]
convert(1000).from('ml').to('l') , I am expecting output as 1.

**Comment by pizzafox** [comment]
The example you provided works

```js
import { convert } from 'convert';

console.log(convert(1000).from('ml').to('l'));
// 1
```

We also have unit tests confirming this behavior:

https://github.com/pizzafox/convert/blob/438bd1ff55db85f87f1ba37b8b913604f8100a95/test/conversions/volume.test.ts#L12

**Comment by yesparthasarathy** [comment]
https://stackblitz.com/edit/angular-ivy-mgcm3z. 

it still shows the wrong output.

**Comment by pizzafox** [comment]
Upgrade to version `v1.1.4`, or a more recent release (`v1.6.2` is the latest at the moment).
This was fixed in 8faa3449d23a907e2fc9e0747c298acbec62b595.
